code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm

```diff
@@ -218,8 +218,16 @@
 	see_in_dark = 8
 	lighting_alpha = LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE
 
+//Tendril-spawned Legion remains, the charred skeletons of those whose bodies sank into laval or fell into chasms.
+/obj/effect/mob_spawn/human/corpse/charredskeleton
+	name = "charred skeletal remains"
+	mob_name = "ashen skeleton"
+	mob_gender = NEUTER
+	husk = FALSE
+	mob_species = /datum/species/skeleton
```

Comment:
These shouldn't be revived with strange reagent, please be careful with adding skeletons without having some kind of additional code to make them basically fake.

> Transplant, insert strange reagent onto some implanted eyes.